docs/release-notes/flink-1.12.md

```diff
@@ -0,0 +1,173 @@
+
+#### Upgrade Mesos version to 1.7 [FLINK-19783](https://issues.apache.org/jira/browse/FLINK-19783)
+
+The Mesos dependency has been bumped from 1.0.1 to 1.7.0. 
+
+#### Send SIGKILL if Flink process doesn't stop after a timeout [FLINK-17470](https://issues.apache.org/jira/browse/FLINK-17470)
+
+In Flink 1.12 we changed the behavior of the standalone scripts to issue a SIGKILL if a SIGTERM did not succeed in shutting down a Flink process. 
+
+#### Introduce non-blocking job submission [FLINK-16866](https://issues.apache.org/jira/browse/FLINK-16866)
+
+The semantics of submitting a job have slightly changed. The submission call returns almost immediately, with the job being in a new `INITIALIZING` state. Operations such as triggering a savepoint or retrieving the full job details are not available while the job is in that state. 
+
+Once the JobManager for that job has been created, the job is in `CREATED` state and all calls are available. 
+
+### Runtime
+
+#### FLIP-141: Intra-Slot Managed Memory Sharing
+
+The configuration `python.fn-execution.buffer.memory.size` and `python.fn-execution.framework.memory.size` have been removed and so will not take effect any more. Besides, the default value of `python.fn-execution.memory.managed` has been changed to true and so managed memory will be used by default for Python workers. In cases where Python UDFs are used together with the RocksDB state backend in streaming or built-in batch algorithms in batch, the user can control how managed memory should be shared between data processing (RocksDB state backend or batch algorithms) and Python, by overwriting [managed memory consumer weights]({% link deployment/memory/mem_setup_tm.md %}#consumer-weights). 
+
+#### FLIP-119 Pipelined Region Scheduling [FLINK-16430](https://issues.apache.org/jira/browse/FLINK-16430)
+
+Beginning from Flink 1.12, jobs will be scheduled in the unit of pipelined regions. A pipelined region is a set of pipelined connected tasks. This means that, for streaming jobs which consist of multiple regions, it no longer waits for all tasks to acquire slots before starting to deploy tasks. Instead, any region can be deployed once it has acquired enough slots for within tasks. For batch jobs, tasks will not be assigned slots and get deployed individually. Instead, a task will be deployed together with all other tasks in the same region, once the region has acquired enough slots. 
+
+The old scheduler can be enabled using the `jobmanager.scheduler.scheduling-strategy: legacy` setting.
+
+#### RocksDB optimizeForPointLookup results in missing time windows [FLINK-17800](https://issues.apache.org/jira/browse/FLINK-17800)
```

Comment:
This feature is first introduced in `1.11`, shall we still need to add this in `1.12-release notes`? The same to Chinese version.